irohad/main/irohad.cpp

```diff
@@ -139,6 +144,22 @@ int main(int argc, char *argv[]) {
       return EXIT_FAILURE;
     }
 
+    // check if ledger data already existing
+    auto ledger_empty = not (
+        irohad.storage->getBlockQuery()->getTopBlockHeight() == 0
+    )
+
+    // Check if force flag to overwrite ledger is specified
+    if (ledger_empty) {
+        if (not FLAGS_overwrite_ledger) {
+            log->error("Block store not empty. Use '--overwrite_ledger' to force
```

Comment:
that isn't compile, consider wrapping both string lines with `"`

Reply:
Hmm, it also possible to just merge fixes from develop. But rebase is also ok

Reply:
Okay I will rebase it then.